hr_expense_cancel/hooks.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
```

Comment:
Put a docstring indicating why we are doing this (for trying to fill the source expense sheet)

Reply:
done